locales/base/translation.json

```diff
@@ -1438,6 +1439,22 @@
       "errorMessageDigitLength": "Account number must be {{length}} digits",
       "errorMessageDigit": "Account number must contain only digits"
     },
+    "mobileMoney": {
+      "operator": {
+        "label": "Network",
+        "placeholderText": "Select a network (ex. T-Mobile)"
+      },
+      "mobile": {
+        "label": "Phone Number",
+        "placeholderText": "Enter number (ex. +00 123 1234)",
```

Comment:
Spaces aren't allowed in this field right? Maybe update the example? Also would be good include a real country code instead of `+00`

Reply:
Updated so now it uses the Nigeria's country code.